src/firefighter/slack/tasks/send_reminders.py

```diff
@@ -116,3 +126,81 @@ def send_reminders() -> None:
                 )
             )
     group(list_fn).apply_async()
+
+
+@shared_task(name="slack.send_reminders_to_commander")
+def send_reminders_to_commander() -> None:
+    # Skip if it's out of office hours
+    if not is_during_office_hours(timezone.localtime()):
+        logger.debug("out of office hours")
+        return
+    # Get all incidents that have reminders (not Gameday, not resolved) and are not ignored
+    opened_incidents = (
+        Incident.objects.filter(
+            _status__gte=IncidentStatus.FIXED.value,
+            _status__lt=IncidentStatus.CLOSED.value,
+            priority__value__lte=5,
+            ignore=False,
+        )
+        .select_related("conversation")
+        .prefetch_related(
+            Prefetch(
+                "roles_set",
+                queryset=IncidentRole.objects.all()
+                .order_by("id")
+                .select_related("role_type", "user", "user__slack_user")
+                .filter(role_type__slug="commander"),
+                to_attr="roles_prefetched",
+            ),
+        )
+    )
+
+    for incident in opened_incidents:
+        try:
+            latest_update = IncidentUpdate.objects.filter(incident=incident).latest(
+                "updated_at"
+            )
+
+            existing_messages_commander = (
+                (
+                    Message.objects.filter(
+                        ff_type=SlackMessageIncidentUpdateReminderCommander.id,
+                        incident=incident,
+                        ts__gte=latest_update.updated_at - timedelta(days=2),
+                    )
+                )
+                .select_related("conversation")
+                .order_by("-ts")
+                .first()
+            )
+
+            existing_messages_channel = (
+                (
+                    Message.objects.filter(
+                        ff_type=SlackMessageChannelReminderPostMortem.id,
+                        incident=incident,
+                    )
+                )
+                .select_related("conversation")
+                .exists()
+            )
+
+            time_since_update = timezone.now() - latest_update.updated_at
+
+            if existing_messages_commander:
+                time_since_last_commander_message = (
+                    timezone.now() - existing_messages_commander.ts
+                )
+                if (
+                    time_since_last_commander_message >= timedelta(days=2)
+                    and not existing_messages_channel
+                ):
+                    send_reminder(incident=incident, to_channel=True)
+
+            if not existing_messages_commander and time_since_update >= timedelta(
+                minutes=5
```

Comment:
5 minutes seem like a debug time. 3 days or more could be appropriate (to avoid sending everything on monday morning if set to 2 days or less)